documents/policies/external/Competition Requirements.md

```diff
@@ -94,96 +115,124 @@ The WCA Delegate must submit the following information when requesting approval
-         2. Exception: A siginificant delay on the previous day, if every affected competitor agrees.
+         2. Exception: A significant delay on the previous day, if every affected competitor agrees.
 9. Championships
    1. A championship type is assigned to competitions that are National, Continental, or World Championships.
       1. A competition must be located entirely within the region where the championship type is selected.
+         1. Exception: A National Championship that only has 3x3x3 Fewest Moves and that is a part of a simultaneous competition, and has at least one location within the region.
       2. A competition may have multiple championship types.
       3. A championship type for a region must only be assigned to one competition in a year.
          1. Exception: Two competitions may have the same Championship status if one of the competitions has only 3x3x3 Fewest Moves as an event and the other competition does not hold the event 3x3x3 Fewest Moves. The two competitions must be held in the same calendar year.
       4. An additional selection process is conducted by the WCA Board for organizing Continental and World Championships.
    2. Naming
       1. Continental and World Championships names shall be as agreed between the Organizing team and the WCA Board.
       2. If used, the word "Championship" must be singular in the competition name.
-      3. Competitions which are not National, Continental, or World championships should avoid names that are similar to championship names.
+      3. Competitions which are not National, Continental, or World Championships should avoid names that are similar to championship names. 
+         1. Competitions with names similar to championship names that are not National, Continental, or World championships should recognize the fastest competitors from the competition’s region.
    3. Date(s)
-      1. National Championships should not be held on the same date(s) as Continental or World championships.
-      2. Continental Championships must not be held on the same date(s) as World Championships.
+      1. National Championships should not be held on the same date(s) as Continental championships of the country’s continent.
+      2. National Championships should not be held on the same date(s) as World championships.
+      3. Continental Championships must not be held on the same date(s) as World Championships.
    4. Championships should have all the officially recognized WCA events.
    5. Championships must recognize and award champion titles and their respective prizes to the winners in the region.
-   6. Organization teams of National Championships should involve all the WCA Delegates of the respective countries, at the discretion of the Senior Delegate.
+   6. Organization teams of National Championships should involve all the WCA Delegates of the respective countries and the regional organization, at the discretion of the Senior Delegate.
```

Comment:
I'd prefer to say "and/or" the regional organization here, since this is how it is applied in practice.